.travis.yml

```diff
@@ -81,31 +69,10 @@ matrix:
     - rvm: jruby-head
       gemfile: gemfiles/postgresql/4-2.gemfile
     - rvm: jruby-head
-      gemfile: gemfiles/postgresql/5-0.gemfile
+      gemfile: gemfiles/postgresql/5-1.gemfile
     - rvm: jruby-head
       gemfile: gemfiles/sqlite3/4-2.gemfile
     - rvm: jruby-head
-      gemfile: gemfiles/sqlite3/5-0.gemfile
+      gemfile: gemfiles/sqlite3/5-1.gemfile
 
-    # legacy testing
-    # things still run and we don't have a good reason to break it
-    # but we don't need to keep running the full matrix
-    - gemfile: gemfiles/mysql/3-0.gemfile
-      rvm: 2.0.0
-    - gemfile: gemfiles/mysql/3-1.gemfile
-      rvm: 2.0.0
-    - gemfile: gemfiles/mysql/3-2.gemfile
-      rvm: 2.0.0
-    - gemfile: gemfiles/postgresql/3-0.gemfile
-      rvm: 2.0.0
-    - gemfile: gemfiles/postgresql/3-1.gemfile
-      rvm: 2.0.0
-    - gemfile: gemfiles/postgresql/3-2.gemfile
-      rvm: 2.0.0
-    - gemfile: gemfiles/sqlite3/3-0.gemfile
-      rvm: 2.0.0
-    - gemfile: gemfiles/sqlite3/3-1.gemfile
-      rvm: 2.0.0
-    - gemfile: gemfiles/sqlite3/3-2.gemfile
-      rvm: 2.0.0
```

Comment:
Yeah, I kept the legacy concept but scoped it to the unsupported AR 4+ versions since only 4.2+ is supported now. I can add AR 3+ back in if you think we should keep it around.